pennylane/workflow/get_gradient_fn.py

```diff
@@ -61,8 +63,10 @@ def _get_gradient_fn(
         )
 
     if diff_method == "best":
-        qn = qml.QNode(lambda: None, device, diff_method=None)
-        return qml.workflow.get_best_diff_method(qn)()
+        if tape and any(isinstance(o, qml.operation.CV) for o in tape):
+            return qml.gradients.param_shift_cv
+
+        return qml.gradients.param_shift
```

Comment:
Why was this changed?

Reply:
The `qml.workflow.get_best_diff_method` function was designed for user convenience, returning the gradient transform as a string. 

So I just took the logic from there and modified it to return the actual transform object instead of the string, aligning more closely with the purpose of `_get_gradient_fn`.

Reply:
Thanks, resolving